studio/components/interfaces/Auth/AuthProvidersForm/ProviderForm.tsx

```diff
@@ -132,6 +144,23 @@ const ProviderForm = ({ config, provider }: ProviderFormProps) => {
                 "
               >
                 <div className="mx-auto my-6 max-w-lg space-y-6">
+                  {provider.title === 'LinkedIn' && (
+                    <Alert_Shadcn_ variant="warning">
+                      <IconAlertTriangle strokeWidth={2} />
+                      <AlertTitle_Shadcn_></AlertTitle_Shadcn_>
```

Comment:
The omission of alert title makes the design of the alert missaligned. Can you please add one?